### PR TITLE
Feature/covid vaccine schema

### PR DESCRIPTION
**Pull request by stephenBarrs**
# New schema
COVID-VACCINE-TRIAL Schema to support fast paced effort for COVID Vaccine Trial sign up form. 

I have  incremented the version in `package.json` to 6.6.0.
department-of-veterans-affairs/va.gov-team#0000

## Pull Requests to update the schema in related repositories

department-of-veterans-affairs/vets-website#0000
department-of-veterans-affairs/vets-api#0000

### CODE REVIEW

File: dist/COVID-VACCINE-TRIAL-schema.json [hunk 1]
```diff
@@ -0,0 +1,330 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Covid Vaccine Trial",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "email": {
```
Comment (mchelen-gov) [inline dist/COVID-VACCINE-TRIAL-schema.json:7]: should this be https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L604 ?

File: dist/COVID-VACCINE-TRIAL-schema.json [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+      "type": "string",
+      "minLength": 6,
+      "maxLength": 80,
+      "format": "email"
+    },
+    "fullName": {
```
Comment (mchelen-gov) [inline dist/COVID-VACCINE-TRIAL-schema.json:13]: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L5 ?

File: dist/COVID-VACCINE-TRIAL-schema.json [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+    "contactHeaderText": {
+      "type": "object",
+      "properties": {
+        "view:contactText": {
+          "type": "object",
+          "properties": {}
+        }
+      }
+    },
+    "zipCode": {
```
Comment (mchelen-gov) [inline dist/COVID-VACCINE-TRIAL-schema.json:219]: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L54

**Comment by mchelen-gov** [comment]
requesting review from @department-of-veterans-affairs/frontend-review-group

File: dist/COVID-VACCINE-TRIAL-schema.json (hunk 1 above)
Comment (stephenBarrs) [reply dist/COVID-VACCINE-TRIAL-schema.json:7]: @mchelen-gov the references are at bottom of file.  email, fullName and date where used from common defs. There may be others that can be used now that I am more familiar with how the schema build process works.

File: dist/COVID-VACCINE-TRIAL-schema.json (hunk 2 above)
Comment (stephenBarrs) [reply dist/COVID-VACCINE-TRIAL-schema.json:13]: @mchelen-gov fullName is from common defs.

File: dist/COVID-VACCINE-TRIAL-schema.json (hunk 3 above)
Comment (stephenBarrs) [reply dist/COVID-VACCINE-TRIAL-schema.json:219]: @mchelen-gov I recall this one giving me a difficult time reusing common def for zipcode without address but I will check it out and see if we can use a common def for zipCode

Comment (stephenBarrs) [reply dist/COVID-VACCINE-TRIAL-schema.json]: @mchelen-gov pushed change to use common def for `usaPostalCode`.  Good catch thanks for the eyes!

File: dist/COVID-VACCINE-TRIAL-schema.json (hunk 1 above)
Comment (stephenBarrs) [reply dist/COVID-VACCINE-TRIAL-schema.json:7]: @rianfowler This is what I am doing, unless I am not understanding correctly.  The common definitions used are linked in at the bottom of `schema.js` and added to the generated .json definitions section. Please let me know if I am not doing that correctly!

File: src/schemas/COVID-VACCINE-TRIAL/schema.js [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+      pattern: '^[1-9]{2}$',
+    },
+    'view:heightInLabel': {
+      type: 'object',
+      properties: {},
+    },
+  },
+};
+const weight = {
+  type: 'string',
```
Comment (mchelen-gov) [inline src/schemas/COVID-VACCINE-TRIAL/schema.js:182]: do we want to limit this to numeric & `.` characters?

**Pull request merged**